_posts/2018-04-25-productivity-with-scripts-screencapture.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "Increasing Productivity with Scripts: Utilizing Mac's screencapture Utility"
+date:   2018-04-25 12:00:00 -0400
+categories: automation scripts
+author: Scott Davidson
+---
+
+I am a firm believer that there is almost always room for improvement in process, especially when it comes to automating tasks. As an Automation Engineer here at Forward Financing, I try to ensure optimal productivity for my team. However, I also constantly look for improvements that I can make within my personal process to help me save time throughout the day.
+
+A great way to accomplish this is by first identifying some cumbersome task, and then writing a script for it. Currently, I have many useful scripts on both my work and home computers to accomplish various things. These scripts help me accomplish tasks in a fraction of the time, saving me minutes or hours in my week. Recently, I encountered an arduous daily process in my workflow that was taking me too much time and wrote a script to help me solve this problem.
```

Comment:
Currently, I have many useful scripts on both my work and home computers to ~accomplish various things. These scripts~ help me accomplish tasks in a fraction of the time